challenges/getFileExtension/solutions/get_file_extension.rb

```diff
@@ -0,0 +1,18 @@
+# Option 1 
+# def get_file_extention(file_name: raise)
```

Comment:
Thanks for your contribution! 🎉  Just a nitpick: `extension` is mistyped as `extention` here and below.